### PR TITLE
PMM-4145 RDS disable metrics flags.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-4145

Ref:
- [x] https://github.com/percona/pmm/pull/500
- [ ] https://github.com/percona/pmm-managed/pull/342
- [ ] https://github.com/percona/grafana-dashboards/pull/629
- [x] https://github.com/percona/rds_exporter/pull/41
- [x] https://github.com/percona/pmm-admin/pull/96

FB:
- [ ]  https://github.com/Percona-Lab/pmm-submodules/pull/772

```
[root@4a969db8134a opt]# pmm-admin inventory add agent rds-exporter --server-url=https://pmm-server  --server-insecure-tls --aws-access-key=*** --aws-secret-key=*** --skip-connection-check pmm-server pmm-server                                                                                                                        
RDS Exporter added.
Agent ID                  : /agent_id/c8e4ec5d-25d1-4f4d-a596-8277d5f167a0
PMM-Agent ID              : pmm-server
Node ID                   : pmm-server
Listen port               : 0

Status                    : <nil>
Disabled                  : false
Disabled Basic Metrics    : false
Disabled Enhanced Metrics : true
Custom labels             : map[]

[root@4a969db8134a opt]# 

```

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

File: commands/inventory/add_agent_rds_exporter.go [hunk 1]
```diff
@@ -24,14 +24,16 @@ import (
...
+PMM-Agent ID              : {{ .Agent.PMMAgentID }}
+Service ID                : {{ .Agent.ServiceID }}
+Listen port               : {{ .Agent.ListenPort }}
 
-Status                : {{ .Agent.Status }}
-Disabled              : {{ .Agent.Disabled }}
-Custom labels         : {{ .Agent.CustomLabels }}
+Status                    : {{ .Agent.Status }}
+Disabled                  : {{ .Agent.Disabled }}
+Disabled Basic Metrics    : {{ .Agent.DisableBasicMetrics }}
```
Comment (BupycHuk) [inline commands/inventory/add_agent_rds_exporter.go]: looks like it should be `.Agent.BasicMetricsDisabled` and `EnhancedMetricsDisabled` here.

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: commands/inventory/add_agent_rds_exporter.go [hunk 2]
```diff
@@ -24,14 +24,16 @@ import (
 
 var addAgentRDSExporterResultT = commands.ParseTemplate(`
 RDS Exporter added.
-Agent ID              : {{ .Agent.AgentID }}
-PMM-Agent ID          : {{ .Agent.PMMAgentID }}
-Service ID            : {{ .Agent.ServiceID }}
-Listen port           : {{ .Agent.ListenPort }}
+Agent ID                  : {{ .Agent.AgentID }}
+PMM-Agent ID              : {{ .Agent.PMMAgentID }}
+Node ID                   : {{ .Agent.NodeID }}
```
Comment (askomorokhov) [inline commands/inventory/add_agent_rds_exporter.go:29]: According to https://github.com/percona/pmm/pull/500/files#diff-d8f7a0a7d4635c8a9d269712db67f417R304

**Review by BupycHuk — APPROVED** [review]

File: commands/inventory/add_agent_rds_exporter.go [hunk 3]
```diff
@@ -93,4 +99,6 @@ func init() {
 	AddAgentRDSExporterC.Flag("aws-secret-key", "AWS Secret Access Key").StringVar(&AddAgentRDSExporter.AWSSecretKey)
 	AddAgentRDSExporterC.Flag("custom-labels", "Custom user-assigned labels").StringVar(&AddAgentRDSExporter.CustomLabels)
 	AddAgentRDSExporterC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddAgentRDSExporter.SkipConnectionCheck)
+	AddAgentRDSExporterC.Flag("disable-basic-metrics", "Disable Basic Metrics").BoolVar(&AddAgentRDSExporter.DisableBasicMetrics)
+	AddAgentRDSExporterC.Flag("disable-enhanced-metrics", "Disable Enhanced Metrics").BoolVar(&AddAgentRDSExporter.DisableEnhancedMetrics)
```
Comment (AlekSi) [inline commands/inventory/add_agent_rds_exporter.go]: We should not capitalise "Metrics" there.

File: commands/inventory/add_agent_rds_exporter.go [hunk 4]
```diff
@@ -24,14 +24,16 @@ import (
...
+Node ID                   : {{ .Agent.NodeID }}
+Listen port               : {{ .Agent.ListenPort }}
 
-Status                : {{ .Agent.Status }}
-Disabled              : {{ .Agent.Disabled }}
-Custom labels         : {{ .Agent.CustomLabels }}
+Status                    : {{ .Agent.Status }}
+Disabled                  : {{ .Agent.Disabled }}
+Disabled Basic Metrics    : {{ .Agent.BasicMetricsDisabled }}
+Disabled Enhanced Metrics : {{ .Agent.EnhancedMetricsDisabled }}
```
Comment (AlekSi) [inline commands/inventory/add_agent_rds_exporter.go]: Enhanced metrics disabled.

**Review by AlekSi — CHANGES_REQUESTED** [review]

**Review by AlekSi — APPROVED** [review]

File: Gopkg.toml [hunk 5]
```diff
@@ -11,7 +11,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-4145-rds-disable-metric-collection"
```
Comment (AlekSi) [inline Gopkg.toml]: Merged

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**